src/utils/init.ts

```diff
@@ -35,13 +35,11 @@ type InitOptions = {
   userId?: string;
   preloadForms?: string[];
   userTracking?: 'cookie' | 'fingerprint';
-  authId?: string;
-  authEmail?: string;
```

Comment:
should we remove authEmail and authPhoneNumber from the backend / db too if we're deprecating them?

Reply:
probably. can you see when the last time either field was set to confirm that no one is using them?